app/boards/shields/qaz/qaz.keymap

```diff
@@ -0,0 +1,75 @@
+#include <behaviors.dtsi>
+#include <dt-bindings/zmk/keys.h>
+#include <dt-bindings/zmk/rgb.h>
+
+#define DEFAULT 0
+#define NUM_SYM 1
+#define NAV     2
+
+/ {
+	behaviors {
+		rst: behavior_tap_hold1 {
+			compatible = "zmk,behavior-tap-hold";
+			label = "Bootloader Tap - Reset Hold";
+			#binding-cells = <0>;
+			hold_ms = <200>;
+			bindings = <&bootloader>, <&reset>, <&kp BKSP>;
+        };
+
+        num_spc: behavior_tap_hold2 {
```

Comment:
Tap hold is not in `main`, so this should not be in here. Once we merge that, this can be restored, but for now this should be in your personal user config keymap for this.